e3nn/point/operations.py

```diff
@@ -3,9 +3,9 @@
 
 
 class Convolution(torch.nn.Module):
-    def __init__(self, Kernel, Rs_in, Rs_out):
+    def __init__(self, kernel):
```

Comment:
Guess I'm going to have to update that tutorial again... If we merge this -- you'll definitely want to make a PSA to the slack.